etc/profile-m-z/rpcs3.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for RPCS3 emulator
+# Description: RPCS3 emulator
+# This file is overwritten after every install/update
+# Persistent local customizations
+include rpcs3.local 
+# Persistent global definitions
+include globals.local # might be a problem
+
+noblacklist ${HOME}/.config/rpcs3
+noblacklist ${HOME}/.cache/rpcs3
+blacklist /usr/libexec
+
+#include disable-common.inc # not working: libstdc++.so.6: version `GLIBCXX_3.4.29' not found
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+#include disable-programs.inc # crashes PPU compilation
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/rpcs3
+mkdir ${HOME}/.config/rpcs3
+whitelist ${HOME}/.cache/rpcs3
+whitelist ${HOME}/.config/rpcs3
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+net none
+netfilter
+nodvd
+nogroups
+#noinput
+nonewprivs
+noroot
+notv
+nou2f
+#novideo
+protocol unix,netlink
+seccomp
+shell none
+tracelog
+
+disable-mnt
+#private-cache
+#private-dev # needs to be disabled for controller support
```

Comment:
Which firejail version do you use?

Reply:
0.9.67

Reply:
And private-dev blocks controllers?

Reply:
I made this assumption based on outdated experience, without testing. I'll remove this comment. I've tested it now, and it seems like there is a bigger issue: games do not seem to start with private-dev. So we should remove it.